Penteract/Assets/Scripts/Fang.cpp

```diff
@@ -203,9 +251,12 @@ void Fang::PlayAnimation() {
 					}
 				}
 			} else {
-				if (compAnimation->GetCurrentState()->name != states[0]) {
+				if (compAnimation->GetCurrentState()->name != states[0] && compAnimation->GetCurrentState()->name != states[21]) {
 					compAnimation->SendTrigger(compAnimation->GetCurrentState()->name + states[0]);
 				}
+				if (compAnimation->GetCurrentState()->name == states[0] && EMP->IsActive()) {
+					compAnimation->SendTrigger(states[0] + states[21]);
```

Comment:
Hardcoded states are a little wierd, this may just be me, we used an enum to make code more readable on OnmaruStates